compute/baremetal.py

```diff
@@ -26,15 +33,52 @@ def __init__(self, **params) -> None:
             root_login
             volumes
             subnet
-            Id
-            location
         """
         # CephVM attributes
         self._roles: list = list()
         self.osd_scenario: Optional[int] = None
         self.keypair: Optional[str] = None
 
         self.params = params
+        self.teuthology_creds = get_cephci_config()["teuthology"]
+        self.conn = None
+
+        # Re-image the node to desired os version
+        self.node = params.get("hostname")
+
+        # Parse the args to get the os details
+        self.os_details = argv[argv.index("--rhel-version") + 1]
+
+        # Parse the teuthology framework related data from config
+        teuthology_infra_node = self.teuthology_creds.get("infra_node")
+        username = self.teuthology_creds.get("username")
+        password = self.teuthology_creds.get("password")
+
+        # Create ssh conn obj
+        r_conn = SSHConnectionManager(teuthology_infra_node, username, password)
+        try:
+            self.conn = r_conn.get_client()
+        except Exception as conn_excep:
+            print(
```

Comment:
Can we have log.error instead of print statement

Reply:
Noted. Will be replaced with log.error